src/components/shortcuts/Skip.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, version 3 of the License.
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+ * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
+ */
+
+/**
+ * Enables one to jump to the end/beginning of a chart.
```

Comment:
Just for the phrasing to be more aligned with the others, I suggest writing something like "Enables the navigation to the end/beginning of a chart"